pkg/util/helper/cronfederatedhpa.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2023 The Karmada Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helper
+
+import (
+	autoscalingv1alpha1 "github.com/karmada-io/karmada/pkg/apis/autoscaling/v1alpha1"
+)
+
+func IsCronFederatedHPARuleSuspend(rule autoscalingv1alpha1.CronFederatedHPARule) bool {
+	if rule.Suspend == nil {
+		return false
+	}
+	return *rule.Suspend
+}
+
+func CronFederatedHPASuccessHistoryLimits(rule autoscalingv1alpha1.CronFederatedHPARule) int {
+	if rule.SuccessfulHistoryLimit == nil {
+		return 3
+	}
+	return int(*rule.SuccessfulHistoryLimit)
+}
+
+func CronFederatedHPAFailedHistoryLimits(rule autoscalingv1alpha1.CronFederatedHPARule) int {
+	if rule.FailedHistoryLimit == nil {
```

Comment:
I have always believed that in the current scenario, there is no need to differentiate between `nil` and `0` for `FailedHistoryLimit` and `SuccessfulHistoryLimit`, so setting them as pointer types is unnecessary.

Reply:
cc @RainbowMango for checking, With implementation,  pointer types are not so convenient.